File: rise_up2/lib/models/material_goods.dart
class MaterialGoods {
  int idMaterialGood;
  String materialGoodName;

  MaterialGoods(
     this.idMaterialGood,
     this.materialGoodName,
  );

  factory MaterialGoods.fromJson(Map<String, dynamic> json) {
    return MaterialGoods(
    json['id_material_good'],
    json['material_good_name']
    );
  }
}

File: rise_up2/Rise_Up_API/Rise_Up_API/Rise_Up_API/Modules/Material_Goods.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Rise_Up_API.Modules
{
    public class Material_Goods
    {
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        [Key] public int id_material_good { get; set; }
        public string material_good_name { get; set; }
    }
}
